### DISCUSSION TITLE
dataV的数据，高德的底图，偏移这么大，高德自己的api就没事

### DISCUSSION THREAD

**Discussion by Kasensill** [antvis/L7#2327] [Q&A]
![099](https://github.com/antvis/L7/assets/126395441/9242b1dd-0268-4b05-914f-fdc71103da6c)

**Comment by Kasensill** [comment]
有没有大佬解释一下这是怎么回事

**Reply by lzxue** [reply]
地图初始化出问题了吧，CSS 样式问题，地图和可视化层没有对应

**Comment by bluseHu** [comment]
坐标系问题吧， 高德的是GCJ02， L7可能是WGS84，可能需要做个转换？
